recipes/yahs/meta.yaml

```diff
@@ -1,21 +1,20 @@
-{% set version = "1.2a" %}
+{% set version = "1.2a.1" %}
 
 package:
   name: yahs
   version: {{ version }}
 
 source:
   url: https://github.com/c-zhou/yahs/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: ddf69943113b99a4371ffa76a5517fa4c699cfbc2449a20cb8fcf6ba0568184a
-  patches:
-    - Makefile.patch
+  sha256: 3dc41d25300a33df02b4b9a99b803eabebb4731148d2b869cc966e0c9f67f13b
 
 build:
-  number: 1
+  number: 0
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - zlib
```

Comment:
it should not be needed in the build section ... maybe the Makefile patch is still needed?